controllers/hostedcontrolplane/hostedcontrolplane.go

```diff
@@ -323,3 +358,145 @@ func (r *HostedControlPlaneReconciler) SetupWithManager(mgr ctrl.Manager) error
 		).
 		Complete(r)
 }
+
+// ------------------------------synthetic-monitoring--------------------------
+
+func (r *HostedControlPlaneReconciler) getDynatraceSecrets(ctx context.Context) (string, string, error) {
+
+	secret := &v1.Secret{}
+	err := r.Client.Get(ctx, types.NamespacedName{Name: secretName, Namespace: secretNamespace}, secret)
```

Comment:
Thoughts on being a little more specific w/ the naming conventions here?
```suggestion
	err := r.Client.Get(ctx, types.NamespacedName{Name: dynatraceSecretName, Namespace: dynatraceSecretNamespace}, secret)
```

I think this would make it easier to add logic to parse an additional secret later, if that ends up becoming necessary

Reply:
Updated, thanks! Though, I don't think hcp reconciler will be parsing additional secrets.